frontend/src/queries/examples.ts

```diff
@@ -49,6 +50,104 @@ const EventsTableFull: DataTableNode = {
     showEventsBufferWarning: true,
 }
 
+const TotalEvents: EventsNode = {
+    kind: NodeKind.EventsNode,
+    properties: [
+        { type: PropertyFilterType.Event, key: '$browser', operator: PropertyOperator.Exact, value: 'Chrome' },
+    ],
+    select: ['total()'],
+}
+
+const TotalEventsTable: DataTableNode = {
+    kind: NodeKind.DataTableNode,
+    source: TotalEvents,
+}
+
+const PropertyFormulas: EventsNode = {
+    kind: NodeKind.EventsNode,
+    properties: [
+        { type: PropertyFilterType.Event, key: '$browser', operator: PropertyOperator.Exact, value: 'Chrome' },
+    ],
+    select: [
+        '1 + 2 + 3',
+        'event',
+        'person.created_at',
+        "concat(properties['$browser'], ' 💚 ', properties['$geoip_city_name']) # Browser 💚 City",
+        "'random string'",
+    ],
+    limit: 100,
+}
+
+const PropertyFormulasTable: DataTableNode = {
+    kind: NodeKind.DataTableNode,
+    source: PropertyFormulas,
+}
+
+const EventAggegations: DataTableNode = {
+    kind: NodeKind.DataTableNode,
+    source: {
+        kind: NodeKind.EventsNode,
+        select: [
+            "concat(properties['$geoip_city_name'], ' ', 'Rocks') # City",
+            'event',
+            'total() + 100000 # Inflamed total',
+            '1 + 2',
+        ],
+        orderBy: ['-total()'],
+    },
+    showReload: true,
+    showEventFilter: true,
+    showPropertyFilter: true,
+    showExport: true,
+    showColumnConfigurator: true,
+}
+
+const DataPerCountry: EventsNode = {
+    kind: NodeKind.EventsNode,
+    properties: [
+        { type: PropertyFilterType.Event, key: '$browser', operator: PropertyOperator.Exact, value: 'Chrome' },
+    ],
+    select: [
```

Comment:
This naming is kind of confusing to me, since `select` (which is a verb) contains properties, and technically properties from `properties` (which is a noun) are selected too. Is this more like `filters` and `fields`?

Reply:
This `select` array is literally the `$THIS` in the `select $THIS from events` query. When this is present, only the columns from here are selected, and nothing else is returned from the API. The `properties` and other filters will have reduced the available set of events for us to play with in an earlier step, so they aren't selected.

Naming things... 😢

I propose to get something in, and then have a RFC to document everything andf clear all of this up.